packages/dataviews/src/dataform-controls/email.tsx

```diff
@@ -32,7 +32,7 @@ export default function Email< Item >( {
 			label={ label }
 			placeholder={ placeholder }
 			value={ value ?? '' }
-			help={ description }
+			help={ help }
```

Comment:
And then this, in order to make it not a breaking change.

```suggestion
			help={ help ?? description }
```